docs/docs-beta/docs/guides/build/external-pipelines/gcp-dataproc-pipeline.md

```diff
@@ -0,0 +1,75 @@
+---
+title: Build pipelines with GCP Dataproc
+description: "Learn to integrate Dagster Pipes with GCP Dataproc to launch external code from Dagster assets."
+sidebar_position: 300
+---
+
+This article covers how to use [Dagster Pipes](/guides/build/external-pipelines/) with [GCP Dataproc](https://cloud.google.com/dataproc) in Job mode.
+
+The [dagster-gcp](/api/python-api/libraries/dagster-gcp) integration library provides the <PyObject section="libraries" object="pipes.PipesDataprocJobClient" module="dagster_gcp" /> resource, which can be used to launch GCP Dataproc jobs from Dagster assets and ops. Dagster can receive regular events such as logs, asset checks, or asset materializations from jobs launched with this client. Using it requires minimal code changes to your Dataproc jobs.
+
+
+<details>
+  <summary>Prerequisites</summary>
+
+    - **In the Dagster environment**, you'll need to:
+
+    - Install the following packages:
+
+        ```shell
+        pip install dagster dagster-webserver 'dagster-gcp[dataproc]'
+        ```
+
+        Refer to the [Dagster installation guide](/getting-started/installation) for more info.
+
+    - **Configure GCP authentication for applications**. If you don't have this set up already, refer to the [GCP authentication guide](https://cloud.google.com/docs/authentication/gcloud).
+
+    - **In GCP**, you'll need:
+
+    - An existing project with a Dataproc cluster.
+    - Prepared infrastructure such as GCS buckets, IAM roles, and other resources required for your Dataproc job.
+
+</details>
+
+## Step 1: Install the dagster-pipes module in your Dataproc environment
+
+Choose one of the [options](https://cloud.google.com/dataproc/docs/tutorials/python-configuration) to install `dagster-pipes` in the Dataproc environment.
+
+For example, use the following property in your configuration:
+
+```yaml
+dataproc:pip.packages: "dagster-pipes,google-cloud-storage"
+```
+
+`google-cloud-storage` is an optional dependency required for passing Pipes messages from the Dataproc job to dagster.
```

Comment:
```suggestion
`google-cloud-storage` is an optional dependency required for passing Pipes messages from the Dataproc job to Dagster.
```

Reply:
The CI was yelling at me for having `Dagster` here - that's why I changed it to lowercase... :melting_face: